oar/core/jira.py

```diff
@@ -232,18 +232,38 @@ def get_high_severity_and_can_drop_issues(self, jira_issue_keys, force):
                     continue
                 else:
                     if issue.is_high_severity_issue():
-                        if force:
-                            if issue.is_cve_tracker():
-                                high_severity_issues.append(key)
-                            else:
-                                can_drop_issues.append(key)
-                        else:
-                            high_severity_issues.append(key)
+                        high_severity_issues.append(key)
                     else:
                         can_drop_issues.append(key)
 
         return high_severity_issues, can_drop_issues
     
+    def get_unverified_issues_except_cve(self, jira_issue_keys):
+        """
+        Get list of CVE issues and list of the other issues that can be dropped by force after confirm droppable message sending.
+
+        Args:
+            jira_issue_keys (list[str]): jira issues keys to be processed
+
+        Returns:
+            tuple[list[str], list[str]]: list of CVE jira keys that are still to be verified, list of jira keys that can be dropped by force.
+        """
+        cve_issues = []
```

Comment:
I think we don't have to return cve issues for this new func. the expected behavior is if the user specify the option `--force`, we'll drop all the unverified issues except cve bugs, and we don't have to warn the user to confirm with bug owner for cve bug dropping, because it cannot be dropped.

Reply:
Yes. Then it will be more make sense. Resolved. Thank you!